src/lib/packager/clients/repositories.rb

```diff
@@ -355,10 +355,8 @@ def repoInfoRichText(name, category, info)
         Ops.get_string(URL.Parse(url), "scheme", "")
       )
       icon_tag = Ops.add(
-        Ops.add(
-          Ops.add(Ops.add("<IMG SRC=\"", Directory.icondir), "/22x22/apps/"),
-          ["cd", "dvd", "iso"].include?(schema) ? "yast-cd_update.png" : "yast-http-server.png"
-        ),
+        Ops.add("<IMG SRC=\"icon://22/\"",
```

Comment:
There is one extra double quote (`"`) at the end.

I think to make it better readable the whole block should be completely rewritten:

```ruby
icon = ["cd", "dvd", "iso"].include?(schema) ? "yast-cd_update.png" : "yast-http-server.png"
icon_tag = "<IMG SRC=\"icon://22/#{icon}\">&nbsp;&nbsp;&nbsp;"
```